packages/vite/src/client/overlay.ts

```diff
@@ -124,7 +140,7 @@ code {
     <pre class="frame" part="frame"></pre>
     <pre class="stack" part="stack"></pre>
     <div class="tip" part="tip">
-      Click outside or fix the code to dismiss.<br>
+      Click outside, press <span class="keyboard-key">Esc</span> key or fix the code to dismiss.<br>
```

Comment:
I think we can use the `kbd` tag here so it's more semantically accurate.

I think this addition would be nice. Perhaps we can tone down the color and bg color of the "Esc" word tag a bit so it doesn't stand out a lot. It's a bit of a vague request but otherwise I could maybe play around a bit and present some options later 😅

Reply:
No worries! I will be responsible for presenting the options!

I will add 3 ~ 4 color options with a screenshot as a comment later this day.

Maybe comeback tomorrow for the review.

Reply:
Awesome thanks. As long as it's not taking too long, that'll be great. I think as long as it blends with the background and doesn't "pop" out, it's fine. It's not worth spending too much time on a small feature 🙌